src/ui/containers/pokemon-hatch-info-container.ts

```diff
@@ -37,77 +37,67 @@ export class PokemonHatchInfoContainer extends PokemonInfoContainer {
     super(x, y);
     this.pokemonListContainer = listContainer;
   }
-  setup(): void {
+
+  setup(): this {
     super.setup();
     super.changeToEggSummaryLayout();
 
-    this.currentPokemonSprite = globalScene.add.sprite(54, 80, "pkmn__sub");
-    this.currentPokemonSprite.setScale(0.8);
-    this.currentPokemonSprite.setPipeline(globalScene.spritePipeline, {
-      tone: [0.0, 0.0, 0.0, 0.0],
-      ignoreTimeTint: true,
-    });
-    this.pokemonListContainer.add(this.currentPokemonSprite);
+    this.currentPokemonSprite = globalScene.add
+      .sprite(54, 80, "pkmn__sub")
+      .setScale(0.8)
+      .setPipeline(globalScene.spritePipeline, {
+        tone: [0.0, 0.0, 0.0, 0.0],
+        ignoreTimeTint: true,
+      });
 
     // setup name and number
-    this.pokemonNumberText = addTextObject(84, 107, "0000", TextStyle.EGG_SUMMARY_DEX, { fontSize: 78 });
-    this.pokemonNumberText.setOrigin(0, 0);
-    this.pokemonListContainer.add(this.pokemonNumberText);
-
-    this.pokemonNameText = addTextObject(7, 109, "", TextStyle.EGG_SUMMARY_NAME, { fontSize: 64 });
-    this.pokemonNameText.setOrigin(0, 0);
-    this.pokemonListContainer.add(this.pokemonNameText);
+    this.pokemonNumberText = addTextObject(84, 107, "0000", TextStyle.EGG_SUMMARY_DEX, { fontSize: 78 }).setOrigin(
+      0,
+      0,
+    );
```

Comment:
```suggestion
    this.pokemonNumberText = addTextObject(84, 107, "0000", TextStyle.EGG_SUMMARY_DEX, { fontSize: 78 }).setOrigin(0);
```
Btw, every `.setOrigin(0, 0)` can be changed to `.setOrigin(0)`.

<img width="1031" height="154" alt="image" src="https://github.com/user-attachments/assets/a0edcc2f-56d3-48e1-8df3-3466e2905083" />